code/game/objects/items/miscellaneous.dm

```diff
@@ -26,3 +26,80 @@
 	user.gib()
 	playsound(src, 'sound/items/eatfood.ogg', 50, 1, -1)
 	return MANUAL_SUICIDE
+
+
+/obj/item/suspiciousphone
+	name = "suspicious phone"
+	desc = "This device raises pink levels to unknown highs."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "suspiciousphone"
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb = list("dumped")
+	var/dumped = FALSE
+	var/mob/living/carbon/human/bogdanoff
+	var/obj/structure/checkoutmachine/checkout
+	var/list/acounts_to_rob
+
+/obj/item/suspiciousphone/attack_self(mob/user)
+	if(!ishuman(user))
+		to_chat(user, "<span class='warning'>This device is too advanced for you!</span>")
+		return
+	if(dumped)
+		to_chat(user, "<span class='warning'>You already activated Protocol CRAB-17.</span>")
+		return FALSE
+
+	if(alert(user, "Are you sure you want to crash this market with no survivors?", "Protocol CRAB-17", "Yes", "No") == "Yes")
+		if(dumped) //Prevents fuckers from cheesing alert
+			return FALSE
+		sound_to_playing_players('sound/items/dump_it.ogg', 75)
+		bogdanoff = user
+		var/turf/targetturf
+		if(GLOB.blobstart.len > 0)
+			targetturf = get_turf(pick(GLOB.blobstart))
+		else
+			targetturf = bogdanoff.loc
+		checkout = new(targetturf)
+		addtimer(CALLBACK(src, .proc/crab17), 100)
+		dumped = TRUE	
+
+/obj/item/suspiciousphone/proc/crab17()
+	acounts_to_rob = SSeconomy.bank_accounts
+	acounts_to_rob -= bogdanoff.get_bank_account()
+	priority_announce("Protocol CRAB-17 has been activated. Get to the credit deposit machine at [get_area(checkout).name] and cash out!", sender_override = "CRAB-17 Protocol")
```

Comment:
this should probably be phrased differently tbh

Reply:
u got any good ideas?